crates/pathfinder/src/ethereum/log/fetch/forward.rs

```diff
@@ -257,12 +257,12 @@ mod tests {
 
         let logs = root_fetcher.fetch(transport.clone()).await.unwrap();
         for log in logs {
-            assert_eq!(log.block_number.0, block_number, "First fetch");
+            assert_eq!(log.block_number.get(), block_number, "First fetch");
```

Comment:
Wondering if it would be nicer to have `block_number` be a `StarknetBlockNumber` instead of u64 with a direct comparison. This would remove the need for the `get()`.

Would require addition to be supported for `StarknetBlockNumber` -- might already be?

Reply:
Aah could had been, it does have the infallible add. I don't think this will save any lines.